<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRouteWindow</class>
 <widget class="QDialog" name="EditRouteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Route</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="routeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>12</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Route:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>13</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Locations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>↑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>↓</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPointButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removePointButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Point</string>
   </property>
  </widget>
  <widget class="QListWidget" name="locationList">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>181</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="routePointList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>181</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dropRouteWindow">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Drop Route</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="routeNameLine">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
